src/request/components/request-detail-panel-logging.jsx

```diff
@@ -45,6 +46,30 @@ function getRowClass(message) {
     return rowClass;
 }
 
+function getDisplayText(level) {
+    return _.startCase(level);
+}
+
+function getIconName(level) {
```

Comment:
To a certain extent, it was an experiment in how easy it would be to incorporate such a library.

Personally, I'd lean towards using a library of web-wide common iconography that does all of the work for me (and makes it easy to manipulate images w.r.t. size, orientation, layout, etc.) rather than create, manage, and manipulate individual, custom images.